cores/esp8266/FS.h

```diff
@@ -179,17 +189,26 @@ class FSConfig
 class SPIFFSConfig : public FSConfig
 {
 public:
-    SPIFFSConfig(bool autoFormat = true) {
+    SPIFFSConfig(bool autoFormat = true, bool enableTime = true) {
```

Comment:
```cpp
class FSConfig
{
public:
  enum fsid { FSId = 0x00000000 };
  FSConfig(bool autoFormat = true, fsid type = FSId)
    : _autoFormat(autoFormat), _type(type) {
  ...
  }
...
};

class SPIFFSConfig : public FSConfig
{
public:
  enum fsid { FSId = 0x53504946 };
  SPIFFSConfig(bool autoFormat = true, bool enableTime = true) 
    : FSConfig(autoFormat, FSId), _enableTime(enableTime) {
  ...
  }
...
};
```

Reply:
Actually, you found a bug not related to style here.  I should remove `enableTime` completely here, since this PR does not include the (massive) SPIFFS changes to allow it to actually store times.